src/lwc/geGiftEntryFormApp/geGiftEntryFormApp.js

```diff
@@ -65,6 +71,22 @@ export default class GeGiftEntryFormApp extends NavigationMixin(LightningElement
         this.namespace = getNamespace(DATA_IMPORT_BATCH_OBJECT.objectApiName);
     }
 
+    connectedCallback() {
+        registerListener('geBatchGiftEntryTableChangeEvent', this.retrieveBatchTotals, this);
+    }
+
+    disconnectedCallback() {
+        unregisterListener('geBatchGiftEntryTableChangeEvent', this.retrieveBatchTotals, this);
+    }
+
+    async retrieveBatchTotals() {
+        this.batchTotals = await BatchTotals(this.batchId);
+
+        if(this.shouldDisplayExpiredAuthorizationWarning) {
```

Comment:
Please see comment for lines 222 - 225.
```suggestion
        if (this.shouldDisplayExpiredAuthorizationWarning()) {
```